### PR TITLE
dk.yml: palmesondag and 1/5 (danish fightday) are not real holidays

### PR DESCRIPTION
**Pull request by bjensen**

### CODE REVIEW

File: lib/generated_definitions/fed_ex.rb [hunk 1]
```diff
@@ -31,7 +31,7 @@ def self.holidays_by_month
   end
 
 def self.day_after_thanksgiving(year)
-  Holidays::DateCalculatorFactory.day_of_month_calculator.call(year, 11, 4, 4) + 1
+  Date.calculate_mday(year, 11, 4, 4) + 1
```
Comment (ptrimble) [inline lib/generated_definitions/fed_ex.rb]: This is a recent change (last two weeks or so) but we do need the fully qualified namespacing here. Can you remove this change? Same with the other cases of this.

**Comment by ptrimble** [comment]
Looks good other than the namespaces. Once that is changed I'll merge. Thanks!

**Comment by bjensen** [comment]
Namespaces? 
I did as the docs instructed. Cloned, did the change, ran the generator and then the tests. All fine. If I did something wrong shouldn't the tests fail and the contribute section be changed to reflect the correct procedure?🙊






On Fri, Jan 8, 2016 at 6:01 PM -0800, "Phil Trimble" <notifications@github.com> wrote:












Looks good other than the namespaces. Once that is changed I'll merge. Thanks!

**Comment by ttwo32** [comment]
\>\> I did as the docs instructed. Cloned, did the change, ran the generator and then the tests. All fine.
You are correct. 
But fed_ex.rb and ups.rb were rerturned when you did 'rake generate'.
The cause was to carelessly forget to commit 2files at https://github.com/holidays/holidays/pull/155.

Because I refactored https://github.com/holidays/holidays/pull/157,  can you merge master to your branch and PR again?

Thanks.

**Comment by ptrimble** [comment]
Yes, @ttwo32 has it. Sorry that I wasn't more clear, my mistake. All we need is for you to rebase to the latest master and we'll be golden.

Again, sorry for the confusion.

**Comment by ptrimble** [comment]
This miscommunication was my fault so I took it upon myself to implement the changes (with the proper attribution). Closing this in favor of https://github.com/holidays/holidays/pull/159

**Pull request closed without merging**